src/main/rx/crsf.c

```diff
@@ -114,6 +116,107 @@ struct crsfPayloadRcChannelsPacked_s {
 
 typedef struct crsfPayloadRcChannelsPacked_s crsfPayloadRcChannelsPacked_t;
 
+#if defined(USE_CRSF_LINK_STATISTICS)
+/*
+* 0x14 Link statistics
+* Payload:
+*
+* uint8_t Uplink RSSI Ant. 1 ( dBm * -1 )
+* uint8_t Uplink RSSI Ant. 2 ( dBm * -1 )
+* uint8_t Uplink Package success rate / Link quality ( % )
+* int8_t Uplink SNR ( db )
+* uint8_t Diversity active antenna ( enum ant. 1 = 0, ant. 2 )
+* uint8_t RF Mode ( enum 4fps = 0 , 50fps, 150hz)
+* uint8_t Uplink TX Power ( enum 0mW = 0, 10mW, 25 mW, 100 mW, 500 mW, 1000 mW, 2000mW )
+* uint8_t Downlink RSSI ( dBm * -1 )
+* uint8_t Downlink package success rate / Link quality ( % )
+* int8_t Downlink SNR ( db )
+* Uplink is the connection from the ground to the UAV and downlink the opposite direction.
+*/
+
+typedef struct crsfPayloadLinkstatistics_s {
+
+    uint8_t Uplink_RSSI_1;
+    uint8_t Uplink_RSSI_2;
+    uint8_t Uplink_Link_quality;
+    uint8_t Uplink_SNR;
+    uint8_t active_antenna;
+    uint8_t RF_Mode;
+    uint8_t Uplink_TX_Power;
+    uint8_t Downlink_RSSI;
+    uint8_t Downlink_Link_quality;
+    uint8_t Downlink_SNR;
+
+}crsfPayloadLinkstatistics_t;
```

Comment:
space before the type definition and newlines around struct members can be removed

Reply:
fixed